compiler/passes/createTaskFunctions.cpp

```diff
@@ -485,7 +485,9 @@ void createTaskFunctions(void) {
 
           markOuterVarsWithIntents(block->byrefVars, uses);
```

Comment:
All 3 of these lines might be able to not run if `block->byrefVars` is null. Your call if you want to find out.